### PR TITLE
Lesson 15

### PR DESCRIPTION
**Pull request by M-fkstv**
Homework in filder lesson-15 Homework

### CODE REVIEW

File: lesson- 15 Homework/homework.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+//     .then((result) => result)
+//     .catch((error) => console.log(error.message));
+//
+//   console.log("res", res);
+// });
+
+/// Second part
+
+imgBtn.addEventListener("click", (event) => {
+  fileEl.click();
```
Comment (almuly) [inline lesson- 15 Homework/homework.js:63]: Please, add event.preventDefault(). It's allow you to achive more user friendly behavior, while user will uplod file

**Comment by almuly** [comment]
Please, resolve conflicts in branch

File: lesson- 15 Homework/homework.js (hunk 1 above)
Comment (M-fkstv) [reply lesson- 15 Homework/homework.js:63]: done

**Comment by M-fkstv** [comment]
resolved conflicts

**Pull request merged**